src/services/EdFi.DataManagementService.Api/Core/Model/No.cs

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Text.Json.Nodes;
+using EdFi.DataManagementService.Api.ApiSchema;
+
+namespace EdFi.DataManagementService.Api.Core.Model;
+
+/// <summary>
+/// Null objects to avoid nullable types when null is irrelevant
```

Comment:
A set of null objects, mostly to simplify pipeline step code by avoiding nullable types in PipelineContext